lms/static/js/fixtures/instructor_dashboard/data_download.html

```diff
@@ -1,9 +1,252 @@
-<div class="issued_certificates">
-      <p>${_("Click to list certificates that are issued for this course:")}</p>
-      <span>
-          <input type="button" name="issued-certificates-list" value="View Certificates Issued" >
-          <input type="button" name="issued-certificates-csv" value="Download CSV of Certificates Issued" >
-      </span>
-      <div class="data-display-table certificate-data-display-table" id="data-issued-certificates-table"></div>
-      <div class="issued-certificates-error request-response-error msg msg-error copy"></div>
-</div>
\ No newline at end of file
+<div class="data-download-container  action-type-container">
```

Comment:
Rather than adding all of the template code in this file, let's add a folder `v2` at the current location and separate out tabs in their respective files and include them here.

Reply:
I synced up on chat and understand that this template is used for js fixtures.